docs/platform/howto/byoc/create-custom-cloud.rst

```diff
@@ -5,40 +5,266 @@ A :doc:`custom cloud </docs/platform/concepts/byoc>` is your own cloud infrastru
+                     }
+                  }
+            }
+         ]
+      }
+
+.. dropdown:: Show permissions required for creating the security groups for bastion and workload nodes
+
+    .. code-block:: bash
+
+      {
+         "Version": "2023-10-04",
+         "Statement":
+         [
+            {
+                  "Sid": "Describe",
+                  "Effect": "Allow",
+                  "Action":
+                  [
+                     "ec2:DescribeSecurityGroups",
+                     "ec2:DescribeSecurityGroupRules",
+                     "ec2:DescribeNetworkInterfaces",
+                     "ec2:DescribeTags"
+                  ],
+                  "Resource":
+                  [
+                     "arn:aws:ec2:<region>:<accountID>:security-group/*"
+                  ],
+                  "Condition":
+                  {
+                     "StringLike":
+                     {
+                        "aws:ResourceTag/aiven_security_group": "cce*-security-group-dmz-*",
+                        "aws:ResourceTag/aiven_security_group": "cce*-security-group-workload-*"
+                     }
+                  }
+            },
+            {
+                  "Sid": "Create",
+                  "Effect": "Allow",
+                  "Action":
+                  [
+                     "ec2:AuthorizeSecurityGroupEgress",
+                     "ec2:AuthorizeSecurityGroupIngress",
+                     "ec2:CreateSecurityGroup",
+                     "ec2:CreateTags",
+                     "tag:TagResources"
+                  ],
+                  "Resource": "*",
+                  "Condition":
+                  {
+                     "StringLike":
+                     {
+                        "aws:ResourceTag/aiven_security_group": "cce*-security-group-dmz-*",
+                        "aws:ResourceTag/aiven_security_group": "cce*-security-group-workload-*"
+                     }
+                  }
+            },
+            {
+                  "Sid": "Delete",
+                  "Effect": "Allow",
+                  "Action":
+                  [
+                     "ec2:RevokeSecurityGroupEgress",
+                     "ec2:RevokeSecurityGroupIngress",
+                     "ec2:DeleteSecurityGroup",
+                     "ec2:DeleteTags"
+                  ],
+                  "Resource": "arn:aws:ec2:<region>:<accountID>:security-group/*",
+                  "Condition":
+                  {
+                     "StringLike":
+                     {
+                        "aws:ResourceTag/aiven_security_group": "cce*-security-group-dmz-*",
+                        "aws:ResourceTag/aiven_security_group": "cce*-security-group-workload-*"
+                     }
+                  }
+            }
+         ]
+      }
+
+.. dropdown:: Show permissions required for creating the ``cce*-iam-role`` role, which has all the required policies
```

Comment:
This art is also not needed as all necessary permissions are already included into the comment above